docs/build.md

```diff
@@ -17,7 +17,26 @@ That will let you build everything as described here.  If you also want to be ab
  * python3-yaml
  * zip
 
+# Create a personal apt repository
+
+Set the apt branch environment variable (you will have to do this every session):
+
+    $ export HOMEWORLD_APT_BRANCH=<username>/<branch>
+
+Do `gpg --list-keys --keyid-format long` to find the key you would like to sign your repository with. Add an entry to `building/signing-keys`:
```

Comment:
Perhaps suggest generating a key, and specifically discourage from using a regular long-lived PGP key?